src/js/components/TaskListComponent.jsx

```diff
@@ -227,6 +227,9 @@ var TaskListComponent = React.createClass({
               <th className="text-center">
                 Logs
               </th>
+              <th className="text-center">
```

Comment:
If we add a new column for each feature, it will bloat the UI and avoid re-use

Reply:
True but you also want the logs and the debugger to be quickly accessible. I can imagine a drop down list somewhere but it would affect the UX, IMHO.
I don't feel like it's overloaded yet. I suggest we reassess for the next feature (if any).